Jenkinsfile2

```diff
@@ -106,6 +106,8 @@ def needsLargerTest
 SOURCE_DIR = "/home/0xdiag"
 TARGET_DIR = "/tmp/pydatatable_large_data"
 
+S3_FOLDER_STABLE = 'stable'
```

Comment:
Can this be replaced with `"s3://h2o-release/datatable/stable"`, so that looking at the top of the file one could see immediately where the releases are being published?